.github/workflows/test.yml

```diff
@@ -21,6 +21,20 @@ jobs:
           version: v1.42.1
           args: --verbose
 
+  yamllint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Install yamllint
```

Comment:
This shouldn't be necessary; all Github runners should have the latest `yamllint` pre-installed (via `pip`): https://github.com/actions/virtual-environments/issues/1657#issuecomment-712228497

Reply:
Not keeping up to speed on the latest github developments, just matching the local environment

Reply:
I don't think it makes a big difference, but the apt version of `yamllint` is typically older than what you get via `pip` install. I don't really care that much; but removing redundant code and potentially having a more up-to-date version seem to favour just deleting this step.

Put let me know if you prefer to merge as-is; we can always change later.